changelogs/fragments/51-add-instance_info-module.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Added instance_info module
```

Comment:
New modules shouldn't have a (manual) changelog entry, they'll automatically have changelog entries generated by `antsibull-changelog`, so a fragment is only needed if there are other changes in the PR that should be reported.